src/Docker.PowerShell/Docker.psd1

```diff
@@ -33,6 +33,7 @@ Copyright = 'Copyright (c) 2016 Microsoft'
 # Cmdlets to export from this module
 CmdletsToExport = @(
     'ConvertTo-ContainerImage',
+    'Copy-ContainerFile',

```

Comment:
Did this sneak in from another commit?  I don't see Copy-ContainerFile implemented in this change...